include/swift/WALASupport/SILWalaInstructionVisitor.h

```diff
@@ -24,136 +24,136 @@ class SILWalaInstructionVisitor : public SILInstructionVisitor<SILWalaInstructio
+  jobject visitRefElementAddrInst(RefElementAddrInst *REAI);
+
+  /*******************************************************************************/
+  /*                          Enums                                              */
+  /*******************************************************************************/
+
+  jobject visitEnumInst(EnumInst *EI);
+  jobject visitUncheckedEnumDataInst(UncheckedEnumDataInst *UED);
+  jobject visitSelectEnumInst(SelectEnumInst *SEI);
+
+  /*******************************************************************************/
+  /*                          Protocol and Protocol Composition Types            */
+  /*******************************************************************************/
+
+  jobject visitAllocExistentialBoxInst(AllocExistentialBoxInst *AEBI);
+  jobject visitProjectExistentialBoxInst(ProjectExistentialBoxInst *PEBI);
+
+  /*******************************************************************************/
+  /*                          Blocks                                             */
+  /*******************************************************************************/
+
+  /*******************************************************************************/
+  /*                          Unchecked Conversions                              */
+  /*******************************************************************************/
+
+  jobject visitAddressToPointerInst(AddressToPointerInst *ATPI);
+  jobject visitThinToThickFunctionInst(ThinToThickFunctionInst *TTFI);
+  jobject visitThinFunctionToPointerInst(ThinFunctionToPointerInst *TFPI);
+  jobject visitConvertFunctionInst(ConvertFunctionInst *CFI);
+
+  /*******************************************************************************/
+  /*                          Checked Conversions                                */
+  /*******************************************************************************/
+
+  /*******************************************************************************/
+  /*                          Runtime Failures                                   */
+  /*******************************************************************************/
+
+  /*******************************************************************************/
+  /*                           Terminators                                       */
+  /*******************************************************************************/
+
+  jobject visitUnreachableInst(UnreachableInst *UI);
+  jobject visitReturnInst(ReturnInst *RI);
+  jobject visitThrowInst(ThrowInst *TI);
+  jobject visitBranchInst(BranchInst *BI);
+  jobject visitCondBranchInst(CondBranchInst *CBI);
+  jobject visitSwitchValueInst(SwitchValueInst *SVI);
+  jobject visitSelectValueInst(SelectValueInst *SVI);
+  jobject visitSwitchEnumInst(SwitchEnumInst *SWI);
+  jobject visitTryApplyInst(TryApplyInst *TAI);
+  jobject visitTupleExtractInst(TupleExtractInst *TEI);
```

Comment:
Not the correct place. Please use the same order as Apple.

Reply:
Can you specifically refer me to some ordering?